docs/compression_algorithms/post_training/Quantization.md

```diff
@@ -0,0 +1,88 @@
+## Post-Training Quantization
+
+Post-Training Quantization is a quantization algorithm that doesn't demand retraining of a quantized model.
+It utilizes a small subset of the initial dataset to calibrate quantization constants.
+
+NNCF provides an advanced Post-Training Quantization algorithm, which consists of the following techniques:
+
+1) MinMaxQuantization - Analyzes the model and inserts extra quantization layers calibrated on a small subset.
+2) FastBiasCorrection or BiasCorrection - Reduces the bias errors between quantized layers and the corresponding
+   original layers.
+
+### Usage
+
+To start the algorithm, provide the following entities:
+
+* Original model.
+* Validation part of the dataset.
+* [Data transformation function](#data-transformation-function) from the original dataset format to the NNCF format.
+
+The basic workflow steps:
+
+1) Create the [data transformation function](#data-transformation-function).
+
+```python
+def transform_fn(data_item):
+    images, _ = data_item
+    return images
+```
+
+2) Initialize NNCF Dataset with a validation dataset and a transformation function.
+
+```python
+calibration_dataset = nncf.Dataset(val_dataset, transform_fn)
+```
+
+3) Run the quantization pipeline.
+
+```python
+quantized_model = nncf.quantize(model, calibration_dataset)
+```
+
+### Data Transformation Function
```

Comment:
I find this section not very clear.

Reply:
This section aims to describe the goal of having transformation_fn and the guidelines how to write it for each backend.